eva/server/command_handler.py

```diff
@@ -32,6 +34,10 @@ def execute_query(query) -> Iterator[Batch]:
     Execute the query and return a result generator.
     """
     stmt = Parser().parse(query)[0]
+    try:
+        StatementBinder(StatementBinderContext()).bind(stmt)
+    except Exception:
+        raise RuntimeError('Invalid Query')
```

Comment:
The binder's exception contains why it fails. Shall we include that instead of only "Invalid Query"?